### ISSUE TITLE
Comma first notation in package.json prevents automated updates

### ISSUE THREAD

**Issue by lennym** [i18next/i18next#373]
Just a niggle... I'm trying to contribute some stuff and keep to the existing code-style but it creates huge diffs when installing modules with `--save` and `--save-dev` flags so makes contributing (*slightly*) more effort as I can't use those tools to automate things.

I'd recommend reverting at least package.json to use npm default notation instead of comma-first to smooth the road for contributors. If you're OK with this then I'll include it in one of my upcoming PRs.

**Comment by jamuhl** [comment]
really ok with that...really feel free to change...most i care about is a) staying compatible or b) have a proper semver and comments in Changelog (releasenotes) mention what's breaking (plus having tests for new stuff).

not sure what your mid/longterm plans are - but if you keep up working on i18next i would be open to give you eg. write rights (honestly i would love to invest more time into i18next - but like most i can only work on this in spare time - where it's always a tradeoff between OSS and family).

**Comment by lennym** [comment]
Excellent. I shall add a PR shortly ;-)

I'll get back to you with a longer answer to the second part...

**Comment by jamuhl** [comment]
thank you..

**Comment by lennym** [comment]
As promised...

I certainly have a vested interest in maintaining at least the node implementation of i18next for at least the duration of this project (likely to be *at least* the next 3 months). I'd be happy to help out with that side.

However, I have never - and am unlikely to ever - implement i18next in the browser, so I'm not certain I have the knowledge to be that effective in terms of reviewing/maintaining changes to that code base beyond pure bug-fixes (like #375).

In short - I'm happy to help with support and PR/issue review, and will do what I can where I am comfortable, but I can't make any promises or commitments to ongoing development.

**Comment by jamuhl** [comment]
sounds fair enough...i really appreciate your effort.
